### DISCUSSION TITLE
Menu position on screen

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#84] [Q&A] [answered]
Do you think the menu could appear at the bottom (left | center | right) of the screen for easier handling with one hand?
Many phones have changed their user interface in this way.

It could be optional via the settings. For example:

<img src="https://user-images.githubusercontent.com/3484020/224785026-44719c88-cc46-4c77-8a30-a8592db1b6d4.jpg" width="50%"/>

**Comment by airald** [comment]
I preffer centered, except if free space can be used to see something else.

Maybe could be interesting have 8 or 10 buttons...

**Reply by DolomitesTom** [reply]
prefer at the bottom

**Reply by ramondevic** [reply]
Centered will be good :)

**Comment by devemux86** [comment]
I can add a setting for the menu position:
- Center (default)
- Bottom

Do we need more options?
- Bottom left
- Bottom right

The menu now takes up the entire width of the screen.
We'll have to change it if want to move it to the corners.

**Reply by umpfmpf** [reply]
For me it looks good like in https://github.com/devemux86/cruiser/discussions/389#discussioncomment-7743708.
Nice would be to have the menu in the bottom half in center with two-sided menu but without the symbols.
Center and Bottom would be a nice option, left and right with this menu size would be completely unnecessary. 
However, all of this just inflates the app and makes it complicated....

**Comment by devemux86** [comment]
<img src="https://github.com/devemux86/cruiser/assets/3484020/668a57aa-72d5-4565-8ec6-b1aceaba00e1" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/39264c78-9320-4bb3-817d-aaf464b29ce6" width="50%"/>

**Reply by umpfmpf** [reply]
I would like it!

**Reply by devemux86** [reply]
Menu position is available for testing in the latest:
- https://github.com/devemux86/cruiser/discussions/390

**Comment by devemux86** [comment]
How about inside navigation? Shouldn't the navigation dialogs always be centered?

<img src="https://github.com/devemux86/cruiser/assets/3484020/2c7f382b-2af2-467e-8443-ba5d42f906e3" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/1dabcb97-cea2-4773-8757-d4ca85f3ecff" width="50%"/>

**Reply by umpfmpf** [reply]
It would be more uniform if these menus can also be moved down. 
However, I would do without swiping upwards when navigating, which could be problematic when operating with the glove.

**Reply by bios4** [reply]
> How about inside navigation? Shouldn't the navigation dialogs always be centered?

I think when someone opens one of these two menus, he's not particularly interested in the very details of the current navigation.
So from my point of view also these two menus should follow the settings which are set in the new "Menu position" option.

**Reply by devemux86** [reply]
Bottom position usually applies when we hold the phone in our hands.
e.g. Samsung calls this "One-handed mode"

When we place the phone in a case in car / motorcycle / bike etc.
Then the center position should be more useful?

**Reply by bios4** [reply]
> When we place the phone in a case in car / motorcycle / bike etc. then the center position should be more useful?

Why?
Where's the difference if the phone is in a holder or not?

"Aiming" with the fingers should be equally easy (or hard)... 😉

**Reply by airald** [reply]
If the phone is on a holder should be more or less the same. The distance from the user's hand could be different but not much...

**Comment by DolomitesTom** [comment]
When we place the phone in a case in car / motorcycle / bike etc.
Then the center position should be more useful?

Yes while navigation centered is better.

**Comment by DolomitesTom** [comment]
Petfect

**Comment by devemux86** [comment, marked as answer]
Bottom / center menu position was implemented in [Cruiser 3.0.37](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
